### PR TITLE
Add option documentation to the readme

### PR DESCRIPTION
**Pull request by JoeChapman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,3 +78,18 @@ checkbox-required
...
 textarea
 ```
+
+## Options
+
+- `className`: A string or array of string class names.
+- `label`: The intended value of the HTML `label` attribute.
+- `type`: The value of the HTML input `type` attribute.
+- `required`: Value applied to `aria-required` HTML attribute.
+- `hint`: Simliar to a `label`. Applicable to text-based fields such as, `textarea`, `input-text` and `input-date`.
```
Comment (gavboulton) [inline README.md]: This adds context to the label, which it is a part of, for input text, radio groups and textarea. It is used within the input by `aria-describedby` for screen readers.

File: README.md [hunk 2]
```diff
@@ -78,3 +78,18 @@ checkbox-required
...
+- `maxlength`: Applicable to text-based fields and mapped to the `maxlength` HTML attribute.
+- `options`: Applicable to HTML `select` and `radio` controls and used to generate the items of either HTML element.
+- `selected`: Applicable to `select`, `checkbox`, and `radio` controls. Will render the selected HTML option/element selected or checked.
+- `legend`: Applicable to `radio` button controls, which are wrapped in a HTML `fieldset` with a `legend` element.
+- `legendClassName`: Applied as a class name to HTML `legend` attribute.
+- `toggle`: Can be used to toggle the display of the HTML element with a matching `id`. See [toolkit](https://github.com/UKHomeOffice/passports-frontend-toolkit/blob/master/assets/javascript/progressive-reveal.js) for details.
```
Comment (gavboulton) [inline README.md]: Can we use "passports-frontend-toolkit" please?

File: README.md [hunk 3]
```diff
@@ -78,3 +78,18 @@ checkbox-required
...
+- `attributes`: A hash of key/value pairs applicable to a HTML `textarea` field. Each key/value is assigned as an attribute of the `textarea`.
```
Comment (gavboulton) [inline README.md]: For example `spellcheck="true"`.

**Comment by gavboulton** [comment]
This is looking good @JoeChapman. I've added a few comments.

**Comment by easternbloc** [comment]
LGTM

**Pull request merged**